src/demo-app/examples/card/fancy/card-fancy-example.html

```diff
@@ -0,0 +1,19 @@
+<md-card>
+  <md-card-header>
+    <div md-card-avatar class="header-image"></div>
+    <md-card-title>Shiba Inu</md-card-title>
+    <md-card-subtitle>Dog Breed</md-card-subtitle>
+  </md-card-header>
+  <img md-card-image src="https://upload.wikimedia.org/wikipedia/commons/7/74/Shiba_Inu_-lying.jpg">
```

Comment:
Not sure sourcing assets from wikimedia directly like this is allowed (especially without attribution)